<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>autozakaz</class>
 <widget class="QWizard" name="autozakaz">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wizard</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QWizardPage" name="wizardPage1">
   <widget class="QCalendarWidget" name="calendarStart">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>264</width>
      <height>156</height>
     </rect>
    </property>
    <property name="selectedDate">
     <date>
      <year>2012</year>
      <month>5</month>
      <day>19</day>
     </date>
    </property>
    <property name="firstDayOfWeek">
     <enum>Qt::Monday</enum>
    </property>
    <property name="gridVisible">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QCalendarWidget::SingleSelection</enum>
    </property>
    <property name="horizontalHeaderFormat">
     <enum>QCalendarWidget::ShortDayNames</enum>
    </property>
    <property name="verticalHeaderFormat">
     <enum>QCalendarWidget::NoVerticalHeader</enum>
    </property>
    <property name="navigationBarVisible">
     <bool>true</bool>
    </property>
    <property name="dateEditEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Рассчитать продажи со следующей даты:</string>
    </property>
   </widget>
  </widget>
  <widget class="QWizardPage" name="wizardPage_6">
   <widget class="QCalendarWidget" name="calendarPost">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>264</width>
      <height>156</height>
     </rect>
    </property>
    <property name="firstDayOfWeek">
     <enum>Qt::Monday</enum>
    </property>
    <property name="gridVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Поставка будет следующего числа:</string>
    </property>
   </widget>
  </widget>
  <widget class="QWizardPage" name="wizardPage2">
   <widget class="QCalendarWidget" name="calendarStop">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>264</width>
      <height>156</height>
     </rect>
    </property>
    <property name="firstDayOfWeek">
     <enum>Qt::Monday</enum>
    </property>
    <property name="gridVisible">
     <bool>true</bool>
    </property>
    <property name="verticalHeaderFormat">
     <enum>QCalendarWidget::NoVerticalHeader</enum>
    </property>
    <property name="navigationBarVisible">
     <bool>true</bool>
    </property>
    <property name="dateEditEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>331</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Товара должно хватить до следующего числа:</string>
    </property>
   </widget>
  </widget>
  <widget class="QWizardPage" name="wizardPage">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>481</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Файл остатков магазина</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Остатки склада</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_2"/>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>Ассортимент обязательный</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_4"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_3"/>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>Количество в коробках</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit_5"/>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="pushButton_5">
       <property name="text">
        <string>Квантование коробок</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QCheckBox" name="check_today_box">
       <property name="text">
        <string>Проверять сегодняшние продажи</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>calendarStart</tabstop>
  <tabstop>calendarStop</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>lineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
